.github/workflows/example-using-reusable-components.yml

```
@@ -0,0 +1,321 @@
+---
+name: 🔄 Example: Using Reusable Components
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 2: mapping values are not allowed in this context

(syntax-check)